config/src/main/java/com/typesafe/config/impl/SimpleConfig.java

```diff
@@ -153,7 +153,7 @@ static private AbstractConfigValue findKeyOrNull(AbstractConfigObject self, Stri
             ConfigValueType expected, Path originalPath) {
         AbstractConfigValue v = self.peekAssumingResolved(key, originalPath);
         if (v == null)
-            throw new ConfigException.Missing(originalPath.render());
+            throw new ConfigException.Missing(self.origin(), originalPath);
```

Comment:
I think I'd rather keep the `Path.render()` here instead of making `Path` public. Otherwise the PR lgtm, thanks!

Reply:
I used `Path` as argument because constructor with string argument ar used in `Null` exception